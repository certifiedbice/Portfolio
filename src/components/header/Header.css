#site-header{
	position: fixed;
	display: block;
	top: 0;
	left: 0;
	z-index: 98;
	width: 100%;
	height: 105px;
	background-color: #fff;
	padding-top: 15px;
	text-align: center;
	font-size: 24px;
	color: #000;
	box-shadow: 0 8px 8px #000;
}
#site-title{
	margin: 0;
	font-size: 24px;
}
#site-description{
	font-family: Verdana, Geneva, Tahoma, sans-serif;
	font-size: 12px;
}
/*---------- # Nav -------------------------------------------*/
nav{
	flex: 0;
	margin: 10px 0 0 0;
	width: 100%;
}

nav ul{
	display: flex;
	flex-direction: row;
	justify-content: center;
	list-style: none;
	padding: 0;
}

nav ul li{
  	flex: 0;
  	margin: 5px;
	font-family: Verdana, Geneva, Tahoma, sans-serif;
}

nav ul li a{
  	font-size: 11px;
  	font-weight: 700;
  	color: #000;
  	text-decoration: none;
	cursor: pointer;
}

nav ul li a:hover{
  color: #555;
}
@media(min-width:700px){
nav ul li a{
  	font-size: 14px;
  	font-weight: 700;
}
}