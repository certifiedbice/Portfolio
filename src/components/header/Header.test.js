import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import Header from './Header';

it('renders without crashing',()=>{
    const header=document.createElement('header');
    ReactDOM.render(<BrowserRouter><Header/></BrowserRouter>,header);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<BrowserRouter><Header/></BrowserRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});