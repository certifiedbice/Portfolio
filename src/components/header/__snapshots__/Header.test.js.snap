// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  id="site-header"
>
  <div
    id="site-info"
  >
    <h1
      id="site-title"
    >
      Portfolio of Bice
    </h1>
    <div
      id="site-description"
    >
      My various projects in all their variousness.
    </div>
    <nav>
      <ul
        id="primary-menu"
      >
        <li>
          <a
            href="/home"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/about"
          >
            About
          </a>
        </li>
        <li>
          <a
            href="/contact"
          >
            Contact
          </a>
        </li>
        <li>
          <a
            href="/portfolio"
          >
            Portfolio
          </a>
        </li>
        <li>
          <a
            href="/resume"
          >
            Resume
          </a>
        </li>
        <li>
          <a
            href="/services"
          >
            Services
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
